src/net/sf/freecol/client/gui/panel/CargoPanel.java

```diff
@@ -34,13 +34,12 @@
 import net.sf.freecol.common.model.StringTemplate;
 import net.sf.freecol.common.model.Unit;
 
-
 /**
  * A panel that holds units and goods that represent Units and cargo
  * that are on board the currently selected ship.
  */
 public class CargoPanel extends FreeColPanel
-    implements DropTarget, PropertyChangeListener {
+        implements DropTarget, PropertyChangeListener {
```

Comment:
I agree that the indentation should be fixed, but changes like these are best included in their own commits/PRs. The reason being, that the actual changes are more difficult to spot when mixed with stylistic changes.

Reply:
My VSCode automatically reformatted the file. 🤦🏼‍♂️
For these kinds of changes, I usually disable whitespace comparison so that I can see actual changes.